test/unit/krane/ejson_secret_provisioner_test.rb

```diff
@@ -113,7 +113,7 @@ def stub_dry_run_validation_request
   end
 
   def stub_server_dry_run_validation_request
-    stub_kubectl_response("apply", "-f", anything, "--server-dry-run", "--output=name",
+    stub_kubectl_response("create", "-f", anything, "--server-dry-run", "--output=name",
```

Comment:
This is the call that won't actually work 😞

Reply:
Noted OK